### PR TITLE
Add alpha taxons to rummager schema

### PR DESCRIPTION
**Pull request by MatMoore**
This will help us highlight popular information tagged to a taxon (eg education).

https://trello.com/c/fG3pFItN/11-show-popular-content-from-education-in-search

Paired with @rubenarakelyan

### CODE REVIEW

**Comment by tijmenb** [comment]
Would it make sense to 1) call it `taxons` and 2) index the `content_id` of the taxon instead of the slug?

File: test/integration/search/search_test.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require "integration_test_helper"
+require "pry-byebug"
```
Comment (malcolmbaig) [inline test/integration/search/search_test.rb]: Probably don't want to leave this here - worth sticking in that test helper above though.

**Comment by MatMoore** [comment]
I think actually the tests are a bit unrealistic here - when we do the tag lookup, we index a hash of title, content id, base path, so using the slug in the tests is a bit misleading.

Happy to go with taxons over alpha_taxons. I was going for consistency with the content store. Are we planning to rename that at some point?

File: test/integration/search/search_test.rb (hunk 1 above)
Comment (MatMoore) [inline test/integration/search/search_test.rb]: whoops. Will move it.

**Comment by tijmenb** [comment]
@MatMoore the link type is `taxons` and the format is `taxon`. Where are you seeing `alpha_taxons`?

**Comment by MatMoore** [comment]
@tijmenb I may have just imagined the whole thing. Changing it to taxons now.

**Comment by MatMoore** [comment]
Discussed offline with @tijmenb:

We're not planning on adding taxons to rummager as documents, since there is a distinction between taxons as a thing within the taxonomy and navigation pages for a taxon (which we don't have yet). We might even remove the base path field from the taxon format to make this clearer.

So the taxon field in rummager shouldn't expand to hashes containing titles and base paths, like we do for specialist sectors, etc. It's just an array of content ids.

I've also renamed everything to use `taxon` throughout.

File: lib/search/presenters/entity_expander.rb [hunk 2]
```diff
@@ -29,6 +29,7 @@ def initialize(registries)
       'world_locations' => :world_locations,
       'specialist_sectors' => :specialist_sectors,
       'people' => :people,
+      'taxons' => :taxons,
```
Comment (tijmenb) [inline lib/search/presenters/entity_expander.rb]: I think this is not necessary anymore.

**Comment by tijmenb** [comment]
👍 apart from https://github.com/alphagov/rummager/pull/681#discussion_r70653352

File: lib/search/presenters/entity_expander.rb (hunk 2 above)
Comment (MatMoore) [inline lib/search/presenters/entity_expander.rb]: Nope, committed that by accident

**Comment by tijmenb** [comment]
👍

**Pull request merged**